### ISSUE TITLE
Add printing styles

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/calendar#136]

**Comment by eppfel** [comment]
👎 for the print button, it's the browsers functionality.

**Comment by tcitworld** [comment]
> 👎 for the print button, it's the browsers functionality.

A number of users don't know about this.

**Comment by Gaming4LifeDE** [comment]
When is this going to be implemented?

**Comment by ChristophWurst** [comment]
The is no ETA at the moment. This feature is not planned nor did anyone point out that they are working on it.

**Comment by IanJSaul** [comment]
Would be very useful - thanks.

**Comment by kaffeeringe** [comment]
I would also take an csv-export, so I can build my own view in Calc or Excel.

**Comment by rokejulianlockhart** [comment]
@eppfel and @tcitworld, per https://github.com/nextcloud/calendar/issues/136#issuecomment-279331332, does a standard API exist for a website to invoke the browser's print dialog?

**Comment by terenc3** [comment]
> @eppfel and @tcitworld, per [#136 (comment)](https://github.com/nextcloud/calendar/issues/136#issuecomment-279331332), does a standard API exist for a website to invoke the browser's print dialog?

Yes https://developer.mozilla.org/en-US/docs/Web/API/Window/print

**Comment by rokejulianlockhart** [comment]
Consequently, the print button could at least for now merely invoke that, and we could consider a custom interface later. Does seem a little useless though, so I think a custom interface is probably the better option since Menu > Print is all that most browsers need to print a page.

**Comment by localhorst** [comment]
What is the status here? Are there any print styles available? Maybe externally, or someone created custom styles for that?
